src/routes/register_startup.js

```diff
@@ -17,6 +17,6 @@ module.exports = (req, res) => {
     if (err) {
       throw err;
     }
-    res.redirect('/')
+    res.redirect('/submitMessage')
```

Comment:
it's also not good UX if the user moves to the submit message page and just stays there and nothing is happens. but i guess @esraajb will inform us of what UX he is thinking about for this page